### DISCUSSION TITLE
Image Getting Transparent

### DISCUSSION THREAD

**Discussion by singhkailash123** [ImageMagick/ImageMagick#6911] [Help]
### ImageMagick version

7.0.9-Q16

### Operating system, version and so on

Windows 10

### Description

Image getting transparent extracting from AI file , using below magick command. (creating jpeg image)

magick convert  -bordercolor white +repage -density 600 -colorspace sRGB  sourceFileName  -alpha remove -resize 1280x720 -quality 90 targetFileName
Actual File Image
![actual image](https://github.com/ImageMagick/ImageMagick/assets/152649232/e615b357-0e17-456a-b98d-60f23f140b38)
After Convert
![image transparent](https://github.com/ImageMagick/ImageMagick/assets/152649232/8f76f0a3-e5a4-4a8b-867a-0bcd3e66ab34)

**Comment by snibgo** [comment]
You have posted a PNG file, not the AI file.

**Reply by singhkailash123** [reply]
@snibgo -I am extracting the image from ai file(sourcefilename) with artboard no and creating jpg. The above actual image is screenshot of ai file.

**Comment by snibgo** [comment]
>  The above actual image is screenshot of ai file.

Yes, I realise that.

If you post your AI file (eg zip it and upload the zip file) I can try to find a solution.
